src/pages/state-channels/index.vue

```diff
@@ -16,9 +16,20 @@
     </template>
   </page-header>
 
-  <state-channels-panel/>
+  <state-channels-panel v-if="!isLoading"/>
+  <loader-panel v-else/>
 </template>
 
 <script setup>
 import { stateChannelsHints } from '@/utils/hints/stateChannelsHints'
+
+const nuxtApp = useNuxtApp()
```

Comment:
I propose to create a composable in `src/composables/useLoading` that will return `isLoading` ref and then reuse it in every component.

Reply:
Cool suggestion, thanks for the tip .) moved there